web/packages/teleport/src/Discover/Desktop/DiscoverDesktops/ServerIcon.tsx

```diff
@@ -22,7 +22,8 @@ const Server = styled.div`
   height: 16px;
   padding: 0 7px;
   box-sizing: border-box;
-  background: #f5e6fe;
+  background: ${({ theme }) =>
+    theme.name === 'light' ? '#6c6c6c' : '#f2e9f7'};
```

Comment:
Eventually we'll have to fix this right? Depending on the theme name will break custom themes.

Reply:
Correct, we will need to remove these instances and define these specific colours in the theme